RxJavaTemplate/buildSrc/src/main/java/Dependencies.kt

```diff
@@ -0,0 +1,139 @@
+class Dependencies {
+
+    object AppModule {
+        val implementations = listOf(
+            // Kotlin stdlib
+            "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${Versions.KOTLIN_VERSION}",
+            "org.jetbrains.kotlin:kotlin-stdlib:${Versions.KOTLIN_VERSION}",
+            // Android components
+            "androidx.core:core-ktx:${Versions.ANDROID_CORE_VERSION}",
```

Comment:
Ex: `androidx.core:core-ktx:${Versions.ANDROID_CORE_VERSION}` is added multiple times in this file.

I wonder if it would be better if we define all dependencies in a separate object class first, then we can choose which dependencies to include based on the module 

With this approach we can reduce duplication, but also keep all the modules synced 

What do you think? 🙏

Reply:
@Tuubz Initially, I did create separate dependencies ex: `const val ANDROIDX_CORE = "androidx.core:core-ktx:${Versions.ANDROID_CORE_VERSION}"` for common usage, but I see that quite hard to maintenance the version of dependencies between modules, once we update the version, it might cause the unnecessary update to other modules 🤔 Agree with you about the point that approach can reduce the duplication.

Reply:
> once we update the version, it might cause the unnecessary update to other modules

Hmm, but that will be the case in the current implementation too, right? 🙏 

If we change `Versions.ANDROID_CORE_VERSION` it will be applied to all modules :still_thinking:

Reply:
@Tuubz I will update to define the dependencies for common usage 😄